analyzer/src/main/kotlin/AnalyzerResultBuilder.kt

```diff
@@ -41,9 +43,8 @@ class AnalyzerResultBuilder {
 
     fun build(excludes: Excludes = Excludes.EMPTY): AnalyzerResult {
         val duplicates = (projects.map { it.toPackage() } + packages).getDuplicates { it.id }
-        require(duplicates.isEmpty()) {
-            "Unable to create the AnalyzerResult as it contains packages and projects with the same ids: " +
-                duplicates.values
+        if (duplicates.isNotEmpty()) {
+            logger.warn {"Unable to create the AnalyzerResult as it contains packages and projects with the same ids: ${duplicates.values}"}
```

Comment:
The log message now does not reflect what you're doing: You say an analyzer result cannot be created, but still create one.

Reply:
Changed log message